### PR TITLE
code for lac benchmark

### PR DESCRIPTION
**Pull request by kinghuin**
### PR types
Function optimization

### PR changes
Models

### Describe
增加新的打印项，支持benchmark的检测

### CODE REVIEW

**Review by ZeyuChen — DISMISSED** [review]
LGTM

File: examples/lexical_analysis/train.py [hunk 1]
```diff
@@ -39,12 +39,13 @@
 parser.add_argument("--epochs", type=int, default=10, help="Corpus iteration num.")
 parser.add_argument("--batch_size", type=int, default=300, help="The number of sequences contained in a mini-batch.")
 parser.add_argument("--max_seq_len", type=int, default=64, help="Number of words of the longest seqence.")
-parser.add_argument("--device", default="gpu", type=str, choices=["cpu", "gpu", "xpu"] ,help="The device to select to train the model, is must be cpu/gpu/xpu.")
+parser.add_argument("--device", default="gpu", type=str, choices=["cpu", "gpu"] ,help="The device to select to train the model, is must be cpu/gpu/xpu.")
```
Comment (wawltor) [inline examples/lexical_analysis/train.py]: is must be cpu/gpu/xpu -> is must be cpu/gpu

Comment (kinghuin) [reply examples/lexical_analysis/train.py]: DONE

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**